SU2_CFD/src/numerics/NEMO/NEMO_sources.cpp

```diff
@@ -296,7 +296,24 @@ CNumerics::ResidualType<> CSource_NEMO::ComputeVibRelaxation(const CConfig *conf
 CNumerics::ResidualType<> CSource_NEMO::ComputeAxisymmetric(const CConfig *config) {
 
   unsigned short iDim, iSpecies, iVar;
-  su2double rho, rhov, vel2, H, yinv;
+  su2double rho, rhov, vel2, H, yinv, T, Tve, qy_ve, Ru, RuSI;
+  su2double *Ds, **GV, ktr, kve;
+
+  Ds = Diffusion_Coeff_i;
```

Comment:
Maybe add a comment here.  Not necessary but I think looks better.   i.e.  "Renaming for conveniences"

Reply:
Fixed.

Reply:
Okay, addressed this.

Reply:
Thanks!